manifests/n/Neovim/nvim/nightly/0.7.0/Neovim.nvim.nightly.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Neovim.Nvim.nightly
+PackageVersion: 0.7.0
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://github.com/neovim/neovim/releases/download/nightly/nvim-win64.msi
+  InstallerSha256: 5B5FC871DFC8CCB01E900F3ADD8C9909A35A6067932ABFD1D9C9E829686EA783
```

Comment:
That's a really good point (sorry, I'm not a windows or winget user). Wouldn't the PR need to get approved each time? I guess we could pin to a specific release instead of latest nightly.

Reply:
Other than that, looks good to me. Just gotta wait for what the pipelines runs say

Reply:
> That's a really good point (sorry, I'm not a windows or winget user, although this is obvious in hindsight). Wouldn't the PR need to get approved each time (so it would still get broken every day until the PR with the updated sha was merged)? I guess we could pin to a specific release instead of latest nightly, or just put this on hold until the stable release is out.

Yes, the PR would need to get merged each time. The way I've seen it done for other packages is to pin it to a specific version, and then when a new nightly build is created remove the old version and add the new version in a single PR.  #55047 is a good example of this. You don't have to remove the old version (assuming that the download link remains valid), but given that nightly builds could quickly accumulate, it's usually best to just keep the last 1-5 versions.

By doing it that way, it will ensure that the link is always valid (even if it is a few hours past the latest release), and then as soon as the new version's PR gets merged in and published, then the update seamlessly becomes available.

Reply:
> Alternatively, is there an option to ignore the hash? I'm reading through the documentation/old issues but can't seem to find anything.

No, there isn't exactly an easy way to ignore the SHA. The hash is required for user safety as it is one of many ways that we validate a package hasn't changed unexpectedly. Users can bypass the check using `--force`, but only if they are not running as admin. Even then, I would still discourage bypassing the check

Reply:
I may hold off for now, we don't minor version nightly releases so it would be incompatible with our current release process (I'd have to add a dummy value 0.7.0-{1,2,3,etc.} or something in a GHA)